main/ertp/guide/math-helpers.md

```diff
@@ -1,19 +1,36 @@
 # MathHelpers
-MathHelpers are the "arithmetic" operations on extents, used for actions like
-addition, subtraction, and greater than or equal to. There are currently three choices for mathHelpers.
+AmountMath uses MathHelpers to do their extent arithmetic. The results are branded, making
+a new amount (which consists of an extent and a brand). MathHelper operations include addition,
+subtraction, equality checking, greater than testing, and working with empty extents.
```

Comment:
I'd say `comparison` rather than `greater than`, simply because the operation is actually greater than or equal, and you don't want objections from pedantics like me.

Reply:
That sounds good to me.